qubes-gui/src/lib.rs

```diff
@@ -164,13 +164,12 @@ macro_rules! enum_const {
         $p: vis enum $n: ident {
             $(
                 $(#[$j: meta])*
-                ($const_name: ident, $variant_name: ident) $(= $e: expr)?
+                $variant_name: ident $(= $e: expr)?
```

Comment:
Does this cause any problems with weird identifier names?  Not really an issue here, but I am curious.

Reply:
I feel like in that case, macro hygiene would save us.

Reply:
Rust doesn’t have hygiene for items yet, sadly.  But this macro will not be used externally so it is fine.